README.md

```diff
@@ -1 +1,35 @@
-# my-portfolio
+# My Portfolio
+
+> This is project project for students in Microverse.
+
+![screenshot](./app_screenshot.jpg)
```

Comment:
- [ ] (Optional) You wrote the word project twice, kindly delete one of them.
- [ ] (Optional) I would like to suggest that you take a screenshot of your portfolio in mobile view and use it as a screenshot rather than the current screenshot that you have.